clients/extension/src/utils/connectedApps.ts

```
@@ -0,0 +1,32 @@
+export const getDappHost = (url?: string) => {
```

Let's avoid returning invalid values. Instead, we could throw an error if we fail to handle the host, allowing the user of the function to manage the error. Additionally, why call this function if we don't have a URL, the argument shouldn't be optional here?

---

Fixed